src/test/mocks.ts

```diff
@@ -91,6 +92,9 @@ export class MockClient extends MockWatcher implements ErdstallClient {
 	async getAccount(_who: Address): Promise<Account> {
 		throw new Error("cannot query accounts on mock clients");
 	}
+	async getNftMetadata(_token: Address, _id: bigint): Promise<NFTMetadata> {
+		throw new Error("cannot query nft data on mock clients");
+	}
```

Comment:
When testing the marketplace server, we might need to inject metadata. So I'd be great if you could put a `metadata: Map<string, NFTMetadata>` into the mock client with a `setMetadata(token, id, NFTMetadata)` setter whose values are then read here.